Scripts/Python/ki/__init__.py

```diff
@@ -4315,6 +4313,18 @@ def BigKIProcessContentList(self, removeInboxStuff=False):
         for removeidx in removeList:
             del self.BKContentList[removeidx]
 
+        if self.BKFolderListOrder[self.BKFolderSelected] == xLocTools.FolderIDToFolderName(PtVaultStandardNodes.kInboxFolder):
+            PIKAFolder = ptVault().getPeopleIKnowAboutFolder()
+            if PIKAFolder and kLimits.MaxRecentPlayerListSize - len(PIKAFolder.getChildNodeRefList()) > 0:
+                PtDebugPrint(f"xKI.BigKIProcessContentList(): {kLimits.MaxRecentPlayerListSize - len(PIKAFolder.getChildNodeRefList())} remaining recents slots.", level=kWarningLevel)
```

Comment:
We should probably only calculate `kLimits.MaxRecentPlayerListSize - len(PIKAFolder.getChildNodeRefList())` once.